src/modules/facility-loan/facility-loan.controller.test.ts

```diff
@@ -91,4 +98,33 @@ describe('FacilityLoanController', () => {
       expect(bundleIdentifierResponse).toStrictEqual({ bundleIdentifier: expectedBundleIdentifier });
     });
   });
+
+  describe('updateLoanExpiryDate', () => {
+    const { updateLoanExpiryDateRequest } = new UpdateLoanGenerator(valueGenerator, new DateStringTransformations()).generate({
+      numberToGenerate: 1,
+      facilityIdentifier,
+      portfolioIdentifier,
+      loanIdentifier,
+    });
+
+    it('calls the facility loan updateLoanExpiryDate method with expected request', async () => {
+      await controller.updateLoanExpiryDate({ loanIdentifier, facilityIdentifier }, updateLoanExpiryDateRequest);
+
+      expect(facilityLoanServiceUpdateLoanExpiryDate).toHaveBeenCalledWith(loanIdentifier, updateLoanExpiryDateRequest);
+    });
+
+    it('calls the facility loan updateLoanExpiryDate method once', async () => {
+      await controller.updateLoanExpiryDate({ loanIdentifier, facilityIdentifier }, updateLoanExpiryDateRequest);
+
+      expect(facilityLoanServiceUpdateLoanExpiryDate).toHaveBeenCalledTimes(1);
+    });
+
+    it('returns the loan identifier from updateLoanExpiryDate', async () => {
+      when(facilityLoanServiceUpdateLoanExpiryDate).calledWith(loanIdentifier, updateLoanExpiryDateRequest).mockResolvedValueOnce(loanIdentifier);
+
+      const loanIdentifierResponse = await controller.updateLoanExpiryDate({ loanIdentifier, facilityIdentifier }, updateLoanExpiryDateRequest);
+
+      expect(loanIdentifierResponse).toStrictEqual({ loanIdentifier });
+    });
```

Comment:
`facilityLoanService.updateLoanExpiryDate` returns `Promise<void>` so I don't think we want to mock that it resolves with the loan id

I think the test name needs updating to make that a bit clearer as well :)